packages/e2e/example/test_driver/failure_e2e_test.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:async';
+
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+Future<void> main() async {
+  test('fails gracefully', () async {
+    final FlutterDriver driver = await FlutterDriver.connect();
+    final String result =
+        await driver.requestData(null, timeout: const Duration(minutes: 1));
+    await driver.close();
+    expect(
+      result,
+      'failure',
+      skip: true, // https://github.com/flutter/flutter/issues/48601
```

Comment:
question: why is skip used?

Reply:
i'm about to issue a fix for it is the idea.